modules/theme-tools/site-logo/inc/functions.php

```diff
@@ -31,6 +31,50 @@ function jetpack_get_site_logo( $show = 'url' ) {
 }
 
 /**
+ * Retrieve an array of the dimensions of the Site Logo.
+ *
+ * @uses Site_Logo::theme_size()
+ * @uses get_option( 'thumbnail_size_w' )
+ * @uses get_option( 'thumbnail_size_h' )
+ * @uses global $_wp_additional_image_sizes;
+ *
+ * @since 3.6.0
+ *
+ * @return array $dimensions {
+ *		An array of dimensions of the Site Logo.
+ *
+ * 		@type string $width Width of the logo in pixels.
+ * 		@type string $height Height of the logo in pixels.
+ * }
+}
```

Comment:
Oops, how did I miss this? :( Removed in 987894f7071f4756a799b3029a4404f5620dc3d6